packages/auth/src/grant/routes.test.ts

```diff
@@ -927,43 +1032,47 @@ describe('Grant Routes', (): void => {
         })
       })
 
-      test('Cannot get grant details without secret', async (): Promise<void> => {
+      test('Cannot get grant details for revoked grant', async (): Promise<void> => {
+        const revokedGrant = await Grant.query().insert({
+          ...generateBaseGrant(),
+          state: GrantState.Revoked
+        })
         const ctx = createContext<GetContext>(
           {
             headers: {
               Accept: 'application/json',
-              'Content-Type': 'application/json'
+              'Content-Type': 'application/json',
+              'x-idp-secret': Config.identityServerSecret
             },
-            url: `/grant/${grant.interactId}/${grant.interactNonce}`,
+            url: `/grant/${revokedGrant.interactId}/${revokedGrant.interactNonce}`,
             method: 'GET'
           },
-          { id: grant.interactId, nonce: grant.interactNonce }
+          { id: revokedGrant.interactId, nonce: revokedGrant.interactNonce }
         )
-
         await expect(
           grantRoutes.interaction.details(ctx)
         ).rejects.toMatchObject({
-          status: 401
+          status: 404
         })
       })
 
-      test('Cannot get grant details for nonexistent grant', async (): Promise<void> => {
+      test('Cannot get grant details without secret', async (): Promise<void> => {
         const ctx = createContext<GetContext>(
           {
             headers: {
               Accept: 'application/json',
-              'Content-Type': 'application/json',
-              'x-idp-secret': Config.identityServerSecret
+              'Content-Type': 'application/json'
             },
             url: `/grant/${grant.interactId}/${grant.interactNonce}`,
             method: 'GET'
           },
-          { id: v4(), nonce: grant.interactNonce }
+          { id: grant.interactId, nonce: grant.interactNonce }
         )
+
         await expect(
           grantRoutes.interaction.details(ctx)
         ).rejects.toMatchObject({
-          status: 404
```

Comment:
This diff is hard to follow but basically I removed a duplicate test (one of https://github.com/interledger/rafiki/blob/main/packages/auth/src/grant/routes.test.ts#L910, https://github.com/interledger/rafiki/blob/main/packages/auth/src/grant/routes.test.ts#L950)
and added the `Cannot get grant details for revoked grant` test.